CHANGES.md

```diff
@@ -122,13 +122,22 @@ conversion to beam schema options. *Remark: Schema aware is still experimental.*
     values as strings) into Python native types that are written to Avro
     (Python's date, datetime types, decimal, etc). For more information
     see https://cloud.google.com/bigquery/docs/loading-data-cloud-storage-avro#avro_conversions.
+* Licenses/notices of third party dependencies are added to Java and Python SDK docker images. ([BEAM-9136](https://issues.apache.org/jira/browse/BEAM-9136))
+    
+    <b>Java SDK images</b>: By default, it checks if the licenses/notices are pullable, instead of pulling the licenses/notices. In order to pull licenses, `docker-pull-licenses` variable should ba passed.
```

Comment:
```suggestion
    <b>Java SDK images</b>: By default, it checks if the licenses/notices are pullable, instead of pulling the licenses/notices. In order to pull licenses, `docker-pull-licenses` variable should be passed.
```